components/details/DetailsBoard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import DetailsList from './DetailsList';
+
+import styles from './Details.module.css';
+export default function DetailsBoard(props) {
+  let allCurriculumData = props.superblockDashboardData;
+  let studentData = props.studentCurriculumData;
+  let titleInfo = props.superblockTitleInfo;
+
+  let mergedDashboardData = [];
+
+  // reshape data to pass into UI
+  allCurriculumData.forEach(superblockData => {
```

Comment:
lines 13-35: Let's remove this  functionality to API processor an a function we utilize inside this file. Something like: updatedAllStudentCurriculumData(email). Then we can call this function and add the arguement for email inside this file.